### ISSUE TITLE
MA0052 is not reported in interpolated string

### ISSUE THREAD

**Issue by drieseng** [meziantou/Meziantou.Analyzer#475]
MA0052 is not reported in combination with interpolated strings, even though using `nameof()` is more efficient there as well.
```c#
public enum Color
{
    Blue,
    Red,
    Yellow,
}

void Sample()
{
    _ = $"Color is {Color.Blue}"); // non compliant
    _ = $"Color is {Color.Blue:G}"); // non compliant
    _ = $"Color is {Color.Blue:g}"); // non compliant

    _ = $"Color is {Color.Blue:D}"); // compliant
    _ = $"Color is {Color.Blue:x}"); // compliant
}
